File: trunk/quests/befallen/Zeek.pl
# Bregna's Big Mistake


sub EVENT_SAY {

   if($text=~/hail/i){
   
   quest::say("'Ah, so many dead to destroy, so little time. What is it you want? You seek the [potions] perhaps?");
   }

   if ($text=~/potions/i){

   quest::say("Give me the four clues.");

   }
   
   
}

sub EVENT_ITEM {


   if(plugin::check_handin(\%itemcount,18651 => 1,18652 => 1 ,18653 =>1 ,18654 =>1 )){ # Bregna's Big Mistake

      quest::summonitem(13984);
      quest::exp(100);
      
}
plugin::return_items(\%itemcount); # return unused item

 }
#END of FILE Zone:grobb  ID:52023 -- Bregna

File: befallen/Zeek.lua
-- items: 18651, 18652, 18653, 18654
function event_say(e)
	if e.message:findi("hail") then
		e.self:Say("Ah, so many dead to destroy, so little time. What is it you want? You seek the [potions] perhaps?");
	elseif e.message:findi("potions") then
		e.self:Say("Give me the four clues.");
	end
end

function event_trade(e)
	local item_lib = require("items");

	if item_lib.check_turn_in(e.trade, {item1 = 18651, item2 = 18652, item3 = 18653, item4 = 18654}) then
		e.other:QuestReward(e.self,0,0,0,0,13984,100); -- Item: Crate of Potions
	end
	item_lib.return_items(e.self, e.other, e.trade)
end
